.circleci/config.yml

```diff
@@ -1,240 +1,240 @@
 version: 2
 jobs:
-  "percy-finalize":
-    docker:
-      - image: percyio/agent
-    steps:
-      - run: percy finalize --all
+    'percy-finalize':
```

Comment:
does this file just get `prettier`? hard to tell from the diff the real change

Reply:
fixed